src/lib/dnssd/Advertiser.h

```diff
@@ -48,6 +49,16 @@ enum class CommissioningMode
     kEnabledEnhanced // Enhanced Commissioning Mode, CM=2 in DNS-SD key/value pairs
 };
 
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+enum class JointFabricMode
+{
+    kAvailable,     // This device is capable of acting as a Joint Fabric Administrator.
+    kAdministrator, // This device is acting as a Joint Fabric Administrator.
+    kAnchor,        // This device is acting as a Joint Fabric Anchor Administrator.
+    kDatastore      // This device is acting as a Joint Fabric Datastore.
```

Comment:
These are not mutually exclusive in the spec, right?  So shouldn't the values here be 1, 2, 4, 8, per spec?

I don't understand how you represent the spec "14" case otherwise.

Reply:
The bit shift was previously handled outside the enum, and the enum represents the bit positions. I've updated this in the latest commit - thanks for the suggestion!